sound/soc/sof/topology.c

```diff
@@ -894,16 +898,19 @@ static int sof_parse_word_tokens(struct snd_soc_component *scomp,
 			found++;
 		}
 	}
+
+	return found;
 }
 
-static int sof_parse_tokens(struct snd_soc_component *scomp,
-			    void *object,
-			    const struct sof_topology_token *tokens,
-			    int count,
-			    struct snd_soc_tplg_vendor_array *array,
-			    int priv_size)
+static int sof_parse_token_sets(struct snd_soc_component *scomp,
+				void *object,
+				const struct sof_topology_token *tokens,
+				int count,
+				struct snd_soc_tplg_vendor_array *array,
+				int priv_size, int sets, int destsize)
```

Comment:
@juimonen destSize is not very intuitive, we should call it objectSize instead.

Reply:
will change.